public/index.html

```diff
@@ -19,6 +19,7 @@
       crossorigin="anonymous"
     />
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+
```

Comment:
This change is unrelated to what this PR is about - probably added by mistake :) We can live with it though ;)